adding_metadata_web_interface.md

```diff
@@ -0,0 +1,20 @@
+## BETYdb web interface
+
+The following sections provide detailed information on how to add new metadata using the web interface.
+
+To access the web interface, go to the web interface for the version of BETYdb that you are using.
+
+The full list of BETYdb web interfaces is in the [BETYdb Technical Documentation](https://pecanproject.github.io/bety-documentation/technical/distributed-instances-of-betydb.html){target="_blank"}. 
+
+Here is a list of the some of the more commonly used public databases running BETYdb software:
+
+| Institution                  | URL                                     | contact | 
+|------------------------------|-----------------------------------------|-------|
+| Energy Biosciences Institute | https://betydb.org                      |kooper@illinois.edu|
```

Comment:
As in the similar table in section 2.2, these external URLs should open in a new tab or window.  Unfortunately, the syntax to do this is much more verbose: you have to use `[URL](URL (repeated!)){target="_blank"}`.  See the file `introduction.md` used for section 2.2.